dist/rpm/apptainer.spec.in

```diff
@@ -125,6 +126,7 @@ BuildRequires: zlib-devel
 %if "%{_target_vendor}" == "suse"
 Requires: squashfs
 Recommends: fakeroot
+Recommends: libsubid-devel
```

Comment:
This should be BuildRecommends

Reply:
Darn, but that's the same issue as on debian; it doesn't exist.

So instead this Recommends should be BuildRequires, but inside another #if that restricts it to tumbleweed or newer.

Reply:
According to https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto that should be
```
%if 0%{?suse_version} > 1600
```

Reply:
Added the logic for suse.